protocols/64a0f9/NGS_prep.ot2.py

```diff
@@ -0,0 +1,250 @@
+from opentrons import labware, instruments, modules, robot
+from otcustomizers import StringSelection
+import math
+
+metadata = {
+    'protocolName': 'Swift Biosciences Swift 25 Turbo NGS Library Prep',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# load modules and labware
+magdeck = modules.load('magdeck', '1')
+mag_plate = labware.load('biorad_96_wellplate_200ul_pcr', '1', share=True)
+elution_plate = labware.load(
+    'biorad_96_wellplate_200ul_pcr', '2', 'elution plate')
+reagent_rack = labware.load(
+    'opentrons_24_aluminumblock_generic_2ml_screwcap',
+    '3',
+    'reagent tuberack (for mastermixes)'
+)
+tempdeck = modules.load('tempdeck', '4')
+temp_plate = labware.load(
+    'biorad_96_wellplate_200ul_pcr',
+    '4',
+    'temperature module plate',
+    share=True
+)
+reagent_res = labware.load(
+    'usascientific_12_reservoir_22ml', '5', 'reagent reservoir')
+tempdeck.set_temperature(4)
+tempdeck.wait_for_temp()
+tips10 = [
+    labware.load('opentrons_96_tiprack_10ul', slot)
+    for slot in ['6', '7', '8']
+]
+tips300 = [
+    labware.load('opentrons_96_tiprack_300ul', slot)
+    for slot in ['9', '10', '11']
+]
+
+
+def run_custom_protocol(
+        number_of_samples: int = 96,
+        p10_single_mount: StringSelection('right', 'left') = 'right',
+        p300_multi_mount: StringSelection('left', 'right') = 'left'
+):
+    # check
+    if number_of_samples > 96 or number_of_samples < 1:
+        raise Exception('Invalid sample number.')
+    if p10_single_mount == p300_multi_mount:
+        raise Exception('Input different mounts for pipettes.')
+
+    # pipettes
+    p10 = instruments.P10_Single(mount=p10_single_mount, tip_racks=tips10)
+    m300 = instruments.P300_Multi(mount=p300_multi_mount, tip_racks=tips300)
+
+    # reagents
+    e_mm = reagent_rack.wells('A1')
+    l_mm = [
+        well for well in reagent_rack.wells(
+            'B1', length=math.ceil(number_of_samples/48))
+    ]
+    pcr_mm = [
+        well for well in reagent_rack.wells(
+            'A2', length=math.ceil(number_of_samples/48))
+    ]
+    beads = reagent_rack.wells(0)
```

Comment:
Are `beads` and `e_mm` supposed to be in the same well? It would be good practice to keep the wells() argument type consistent in the same protocol, either using names or indices but not both

Reply:
mistake with beads- was supposed to be in reservoir (made edit). I usually use integer indexes the reagent reservoir because there's only one row, but it does look funny juxtaposed with tiprack well names. I'll change for consistency.